code/game/objects/obj_defense.dm

```diff
@@ -28,8 +28,6 @@
 		else
 			return 0
 	var/armor_protection = 0
-	if(damage_flag)
```

Comment:
Undocumented regression. This is not part of our linear armor tier system, this is part of stock TG/Cit. Why is it being removed?